fuse/dl/models/model_wrapper.py

```diff
@@ -31,10 +31,10 @@ class ModelWrapper(torch.nn.Module):
 
     def __init__(self,
```

Comment:
Maybe we should follow the dataset naming (ModelWrapSeqToDict)?

Reply:
renamed to ModelWrapSeqToDict - did not change the file name as we might include additional wrappers there in the future